src/multiplot.inp

```diff
@@ -143,27 +155,56 @@ function string multiplot (const strings files_input "Input file paths/names",
 
     check_os_and_gretl_version()
     check_version_of_extra_package()
+    
+    if exists(options)
+        options = options + get_default_parameters()
+    else
+        bundle options = get_default_parameters()
+    endif
+    
+    # v0.3: Determine whether files_input contains just the file names / paths,
+    #  or already the actual gnuplot command contents.
+    #  Our heuristic is to check for line breaks in the strings, as those should 
+    #  not occur in file names.
+    #  Alternatively, the user can force interpretation with the new flag
+    #  'notempfiles=TRUE' in the options bundle.
+    
+    # check for explicit user spec
+    if inbundle(options, "notempfiles")
+        scalar notempfiles = options.notempfiles 
+    else	# or use heuristic (multi_instrings also from extra)
+        scalar notempfiles = nelem(multi_instrings(files_input, sprintf("\n"))) > 0
```

Comment:
Brauchen wir wirklich multi_instrings() dafür? Es müsste doch via built-in instrings() Funktion reichen, auf den String `\n` zu prüfen, oder?

Reply:
Ah, ich glaube, Du hast Recht. Ich hatte vergessen, dass es als Zwischenlösung von instring() und multi_instrings() auch einfach instrings() gibt!

Reply:
OK, nach Allins Hinweis ist mir klar, dass beides falsch ist. Wir wollen ja nicht auf Gleichheit mit "\n" testen, sondern ob es enthalten ist. Eine einfache Lösung ist natürlich eine Schleife - übrigens testet die native Funktion nlines() auch genau auf newlines "\n", insofern wäre das sogar noch einfacher als instring(). (Aber mit Schleife trotzdem.)

Reply:
Haha, witzig, auf `nlines()` bin ich gestern auch gestoßen  ;-)
Ich arbeite gerade an einer Überarbeitung, die ich aber noch nicht gepusht habe. Sollte in 1-2 Tagen kommen.

Reply:
Habe jetzt doch eine Lösung mit nlines ohne explizite Schleife gefunden. Ist gerade gepusht. Hoffe, dass ich Dir nicht zu sehr in die Quere gekommen bin. Tests stehen noch aus.

Reply:
So, habe gerade die Sache fertig zum Mergen gemacht. D.h. es gibt jetzt ein Beispiel für die neue Funktionalität im sample script, musste dafür natürlich gleich ein paar kleine Bugs beseitigen. Funktioniert jetzt aber eigentlich wie gedacht.
Soll ich das reviewen & mergen vollziehen (kann ich das überhaupt?) oder machst Du das?

Reply:
Ach so: Es gibt ja auch noch einen anderen Branch, der irgendwie 2 Jahre alt zu sein scheint?

Reply:
> Habe jetzt doch eine Lösung mit nlines ohne explizite Schleife gefunden. Ist gerade gepusht. Hoffe, dass ich Dir nicht zu sehr in die Quere gekommen bin. Tests stehen noch aus.

Super, das funktioniert

Reply:
> So, habe gerade die Sache fertig zum Mergen gemacht. D.h. es gibt jetzt ein Beispiel für die neue Funktionalität im sample script, musste dafür natürlich gleich ein paar kleine Bugs beseitigen. Funktioniert jetzt aber eigentlich wie gedacht.

Ich hatte bereits ein bisschen den Code refactored, jedoch bisher nicht gepusht. Ich muss da noch ein Problem lösen und dann können wir das nach master bringen.

> Soll ich das reviewen & mergen vollziehen (kann ich das überhaupt?) oder machst Du das?

Das Review plus der merge funktionieren so:
1. Du reviewst und akzeptiertst irgendwann
2. Du oder ich können dann final den "merge" Button klicken

Reply:
> Ach so: Es gibt ja auch noch einen anderen Branch, der irgendwie 2 Jahre alt zu sein scheint?

Ja, den muss ich mir noch mal anschauen und dann entscheiden, ob der noch relevant ist.

Reply:
> Ich hatte bereits ein bisschen den Code refactored, jedoch bisher nicht gepusht. Ich muss da noch ein Problem lösen und dann können wir das nach master bringen.

Naja, man kann das mit dem refactoring auch übertreiben.... ;-)

Reply:
Das stimmt. Aber ich glaube, gerade die `multiplot()` Funktion ist nun deutlich übersichtlicher, was uns die Arbeit für`s nächste Mal (hoffentlich) erleichtert.

Ich habe gerade den neuesten Stand gepusht. Die Tests und das Beispielskript laufen hier wie erwartet.

Reply:
> Ja, den muss ich mir noch mal anschauen und dann entscheiden, ob der noch relevant ist.

Den 3. alten Branch habe ich gelöscht.

Reply:
@schreibe ist noch irgendetwas offen hier? Falls nicht, dann könnte ich die neue Version auf den gretl zur Prüfung durch die Anderen hochladen. Falls noch Kommentare kommen, kann ich diese in diesem Zweig unterbringen, bevor wir das nach master bringen.

Reply:
Aus meiner Sicht ist nichts offen. Ich würde auch das mergen nicht von den anderen abhängig machen. Falls Kommentare kommen, können wir ja dafür einfach dann einen neuen Branch aufmachen. Aber wie Du willst.